h2o-core/src/main/java/hex/ModelMetricsBinomial.java

```diff
@@ -270,12 +270,13 @@ private ModelMetrics makeModelMetrics(Model m, Frame f, GainsLift gl) {
 
     private GainsLift calculateGainsLift(Model m, Frame preds, Vec resp, Vec weights) {
       GainsLift gl = null;
-      try {
-        gl = new GainsLift(preds.lastVec(), resp, weights);
-        gl.exec(m != null ? m._output._job : null);
-      } catch(Throwable t) { // TODO: Why do we need to catch Throwable here?
-        Log.debug("Calculating Gains-Lift failed", t);
+      gl = new GainsLift(preds.lastVec(), resp, weights);
+      if (m != null && m._parms._gainslift_bins != 0) {
```

Comment:
it looks like there is no default (0), does this mean GL are not calculated anymore?

Reply:
The default is 0, as the parameter is int and that is by default 0 - if unset, it defaults to 0. The internal logic inside `GainsLift.java` is already wrapped around 0 as default.

Reply:
@Pscheidl sorry, that was my misunderstanding, I didn't notice `gl.exec` is outside of the if-statement. Thanks for explaining.

Reply:
No problem at all !